tests/framework/e2e/etcd_process.go

```diff
@@ -183,6 +184,17 @@ func (ep *EtcdServerProcess) Logs() LogsExpect {
 	return ep.proc
 }
 
+func (cfg *EtcdServerProcessConfig) SetInitialCluster(nodes []string, initialClusterState string) {
+	cfg.InitialCluster = strings.Join(nodes, ",")
+	cfg.Args = append(cfg.Args, "--initial-cluster", cfg.InitialCluster)
+	cfg.Args = append(cfg.Args, "--initial-cluster-state", initialClusterState)
+}
+
+func (cfg *EtcdServerProcessConfig) EnableDiscovery(token string, endpoints []string) {
+	cfg.Args = append(cfg.Args, fmt.Sprintf("--discovery-token=%s", token))
+	cfg.Args = append(cfg.Args, fmt.Sprintf("--discovery-endpoints=%s", strings.Join(endpoints, ",")))
+}
```

Comment:
These two methods are only used by `SetInitialOrDiscovery`, it seems that we can just get the code included in it directly instead of creating two separate methods.